packages/react-components/src/echarts/extensions/minMaxDataStreamValues/minMaxDataStreamSync.ts

```diff
@@ -0,0 +1,19 @@
+import SeriesModel from 'echarts/types/src/model/Series';
+import { SeriesOption, DefaultStatesMixin } from 'echarts/types/src/util/types';
+
+export const findMinMax = (
+  series: SeriesModel<SeriesOption<unknown, DefaultStatesMixin>>
+) => {
+  let max: number | undefined = undefined;
+  let min: number | undefined = undefined;
+  series.getData().each((dims) => {
+    const dataPoint = series.getData().getValues(dims).at(1) as number; // get y value
```

Comment:
do we not need to check whether or not the point is contained within the coordinate system? Have you tested all of the different chart types? If we don't need to do this because of our usage with dataZoom, we should note that this function only works because we are filtering data points via dataZoom